dags/vz_google_sheets.py

```diff
@@ -113,39 +124,17 @@ def custom_fail_slack_alert(context: dict) -> str:
     )
 }
 
-dag = DAG(dag_id = dag_name, default_args = DEFAULT_ARGS, schedule_interval = '@daily', catchup = False)
-
-task1 = PythonOperator(
-    task_id='2018',
-    python_callable=pull_from_sheet,
-    dag=dag,
-    op_args=[con, service, 2018, sheets[2018]]
-    )
- 
-task2 = PythonOperator(
-    task_id='2019',
-    python_callable=pull_from_sheet,
-    dag=dag,
-    op_args=[con, service, 2019, sheets[2019]]
-    )
-     
-task3 = PythonOperator(
-    task_id='2020',
-    python_callable=pull_from_sheet,
-    dag=dag,
-    op_args=[con, service, 2020, sheets[2020]]
-    )
-    
-task4 = PythonOperator(
-    task_id='2021',
-    python_callable=pull_from_sheet,
-    dag=dag,
-    op_args=[con, service, 2021, sheets[2021]]
-    )
-
-task5 = PythonOperator(
-    task_id='2022',
-    python_callable=pull_from_sheet,
-    dag=dag,
-    op_args=[con, service, 2022, sheets[2022]]
-    )
\ No newline at end of file
+with DAG(
+    dag_id=dag_name,
+    default_args=DEFAULT_ARGS,
+    schedule_interval="@daily",
+    tags=["Vision Zero"],
```

Comment:
love the tag!

Reply:
It might make it easier to browse the DAGs